cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
@frrrances Studio is using mako templates but sudo template is rendered in django templates and we don't have a main layout in django templates for studio but we have it for LMS, that's why I added this modal after discussing it with @explorerleslie and @cpennington.

We can't add studio header/footer.